libdnf/transaction/CompsGroupItem.hpp

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (C) 2017-2018 Red Hat, Inc.
+ *
+ * Licensed under the GNU Lesser General Public License Version 2.1
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+ */
+
+#ifndef LIBDNF_TRANSACTION_COMPSGROUPITEM_HPP
+#define LIBDNF_TRANSACTION_COMPSGROUPITEM_HPP
+
+#include <memory>
+#include <vector>
+
+namespace libdnf {
+
+enum class CompsPackageType : int {
+    MANDATORY = 0,
+    DEFAULT = 1,
+    OPTIONAL = 2,
+    CONDITIONAL = 3,
+};
```

Comment:
@dmach this is not consistent with libdnf
see https://github.com/rpm-software-management/dnf/blob/master/dnf/comps.py#L47